demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -206,6 +206,7 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
           try {
             chimeMeetingId = await this.authenticate();
           } catch (error) {
+            console.error(error);
```

Comment:
Remove the `console.error`

Reply:
I understand now. The following code shows the same message "UserMedia is not allowed in HTTP sites" for other erros too. Let's keep this `console.error` for now now to miss other errors.